apps/smart-contracts/token/test/Blocklist.test.ts

```diff
@@ -148,4 +160,75 @@ describe('=> Blocklist', () => {
       }
     })
   })
+
+  describe('# reset', () => {
+    beforeEach(async () => {
+      await setupBlocklist()
+      await blocklist.connect(owner).set(blockedUsersArray, blockedArray)
+    })
+
+    it('reverts if not owner', async () => {
+      expect(await blocklist.owner()).to.not.eq(blockedUser1.address)
+
+      await expect(blocklist.connect(blockedUser1).reset(blockedUsersArray)).revertedWith(
+        'Ownable: caller is not the owner'
+      )
+    })
+
+    it('increments blocked accounts index by 1 if no new blocked accounts', async () => {
+      const blockedAccountIndex = await blocklist.getBlockedAllountsIndex()
+
+      await blocklist.connect(owner).reset([])
+
+      expect(await blocklist.getBlockedAllountsIndex()).to.eq(blockedAccountIndex.add(1))
+    })
+
+    it('increments blocked accounts index by 1 if new blocked accounts', async () => {
+      const blockedAccountIndex = await blocklist.getBlockedAllountsIndex()
+
+      await blocklist.connect(owner).reset(newBlockedUsersArray)
+
+      expect(await blocklist.getBlockedAllountsIndex()).to.eq(blockedAccountIndex.add(1))
+    })
+
+    it('clears blocklist if no new blocked accounts', async () => {
```

Comment:
```suggestion
    it('clears list if not setting new list', async () => {
```
Don't need to say blocklist as this is already implied